script/convert-grammars

```diff
@@ -42,7 +42,7 @@ class DirectoryPackage
       case File.extname(path.downcase)
       when '.plist'
         path.split('/')[-2] == 'Syntaxes'
-      when '.tmlanguage'
+      when '.tmlanguage', '.yaml-tmlanguage', '.sublime-syntax'
```

Comment:
Sublime Text YAML syntax definitions use the `.YAML-tmLanguage` file extension, and more recently, `.sublime-syntax`.